django/template/backends/django.py

```diff
@@ -90,3 +105,48 @@ def reraise(exc, backend):
     if hasattr(exc, 'template_debug'):
         new.template_debug = exc.template_debug
     six.reraise(exc.__class__, new, sys.exc_info()[2])
+
+
+def get_installed_libraries():
+    """
+    Return template tag libraries defined in Django and installed applications.
```

Comment:
"Return the built-in template tag libraries and those from installed applications." (may be clearer than "defined in Django")?